src/domain/hanke/edit/HankeFormYhteystiedot.tsx

```diff
@@ -377,6 +386,7 @@ const HankeFormYhteystiedot: React.FC<FormProps> = () => {
         language={locale}
         headingLevel={3}
         heading={t('form:yhteystiedot:titles:otherInfo')}
+        initiallyOpen={Array.isArray(muutTahot) && muutTahot.length > 0}
```

Comment:
Is the `Array.isArray` check needed here? Can `muutTahot` be anything else than array?

Reply:
Removed